pkgs/games/domination/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv
+, fetchsvn
+# jdk8 is needed for building, but the game runs on newer jres as well
+, jdk8
+, jre
+, ant
+, makeWrapper
+, makeDesktopItem
+}:
+
+let
+  desktopItem = makeDesktopItem {
+    type = "Application";
+    name = "Domination";
+    desktopName = "Domination";
+    exec = "domination";
+    icon = "domination";
+  };
+  editorDesktopItem = makeDesktopItem {
+    type = "Application";
+    name = "Domination Map Editor";
+    desktopName = "Domination Map Editor";
+    exec = "domination-map-editor";
+    icon = "domination";
+  };
+
+in stdenv.mkDerivation {
+  pname = "domination";
+  version = "1.2.3";
+
+  # The .zip releases do not contain the build.xml file
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/domination/code/Domination";
+    rev = "1964";
```

Comment:
Where can you find this number? How is this going to work with nixpkgs-update? What happens if it bumps version?

Reply:
Right, I should add a comment. The repo doesn't seem to have tags, so the update is manual unfortunately. nixpkgs-updater will not work (it checks that the package actually changes iirc)

Reply:
Added a comment